libs/libc/stdio/lib_dtoa_engine.c

```diff
@@ -69,7 +83,7 @@ int __dtoa_engine(double x, FAR struct dtoa_s *dtoa, int max_digits,
   uint8_t flags = 0;
   int i;
 
-  if (__builtin_signbit(x))
```

Comment:
I think this was a part of a bugfix recently (not sure if bugfix was correct on usage GNU built-in). I will search for PR